ipatests/test_integration/test_installation.py

```diff
@@ -196,3 +197,36 @@ def test_install_master(self):
 
     def test_install_kra(self):
         tasks.install_kra(self.master, first_instance=True)
+
+
+@pytest.mark.ds_acceptance
```

Comment:
marking TestSimpleReplication: You're right, I don't know why I persuaded myself that it does something else, I'll change that.

extending with ca: No, as was agreed before we don't want to involve CS more that necessary to minimize the risk of failing due to bug other than in DS.